leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -1007,4 +1007,87 @@ private void closeKeyboard() {
       imm.hideSoftInputFromWindow(view.getWindowToken(), 0);
     }
   }
+
+  public static class Builder {
+    private Double locationLatitude;
+    private Double locationLongitude;
+    private String locationSearchZone;
+    private String layoutsToHide = "";
+    private boolean enableSatelliteView = true;
+    private boolean shouldReturnOkOnBackPressed = false;
+    private List<LekuPoi> lekuPois;
+
+    public Builder() {
+    }
+
+    public Builder withLatitude(double latitude) {
```

Comment:
location for builder should be a simple `withLocation()`... why?

Lib cannot work without both params, so it's weird to set `location` or `longitude` ...